tools/rdm/TestMixins.py

```diff
@@ -907,3 +976,49 @@ def VerifyResult(self, response, fields):
            self.DESCRIPTION_FIELD,
            self.current_item,
            fields[self.DESCRIPTION_FIELD].encode('string-escape')))
+
+
+class GetSettingDescriptionsMixinRange(GetSettingDescriptionsMixin):
+  """Perform a GET for each setting in a range.
+
+    The range is a count, it will check FIRST_INDEX_OFFSET to
+    FIRST_INDEX_OFFSET + NumberOfSettings().
+
+    Subclasses must define EXPECTED_FIELD, which is the field to validate the
+    index against and DESCRIPTION_FIELD, which is the field to check for
+    unprintable characters.
+
+    If ALLOWED_NACKS is non-empty, this adds a custom NackGetResult to the list of
```

Comment:
Remove the parts of the comment that are covered in the base class

e.g. Subclasses must define EXPECTED_FIELD,...
If ALLOWED_NACKS is non-empty, ....